src/coreclr/pal/inc/pal.h

```diff
@@ -3722,7 +3722,7 @@ VOID
 PALAPI
 FlushProcessWriteBuffers();
 
-typedef void (*PAL_ActivationFunction)(CONTEXT *context);
+typedef void (*PAL_ActivationFunction)(CONTEXT *context, bool fromDebugger);
```

Comment:
A nit - can you please keep this function pointer unchanged and add the following function instead?
```c++
void HandleSuspensionForInterruptedThread(CONTEXT *interruptedContext)
{
    HandleSuspensionForInterruptedThread(interruptedContext, false);
}
```

Reply:
done